velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,15 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns UserError status if the date is invalid.
+Status daysSinceEpochFromWeekOfMonthDate(
+    int32_t year,
+    int32_t month,
+    int32_t weekOfMonth,
+    int32_t dayOfWeek,
+    int64_t& out);
```

Comment:
consider using velox::Expected as return type instead of Status + output parameter

Reply:
Refactored using velox::Expected in 555e1d9782aec5b6e1c4c22733459922e2c16ad6. Thanks.